Week2/homework/maartjes-work.js

```diff
@@ -45,21 +45,28 @@ const tuesday = [
 const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
-function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
-}
-
-// eslint-disable-next-line no-unused-vars
-const earnings = computeEarnings(maartjesTasks, maartjesHourlyRate);
+const convertDurationsToHour = duties =>
+  duties.map(item => ({
+    name: item.name,
+    duration: item.duration / 60,
+  }));
+const filterTasks = (tasks = []) => tasks.filter(item => item.duration >= 2);
```

Comment:
Nice use of a default parameter to avoid a potential error from being thrown!

You can also think about what happens if `duties` is undefined in `convertDurationsToHour()`. How would you deal with that case?